### PR TITLE
Fix #15667: RTL languages break mixer numbers

### LINKED ISSUE #15667: [MU4 Issue] RTL Languages Break Mixer Numbers
**Describe the bug**
When using any RTL language, the Mixer numbers appear flipped along the side closest to the center, compared to LTR languages

**To Reproduce**
1. Change the app language to any RTL one
2. Open any score
3. Open the Mixer
4. Observe Mixer dB numbers appearing behind Mixer lines

**Expected behavior**
Mixer appears normally no matter the language.

**Screenshots**
Mixer 
image

**Platform information**
 - OS: Windows 10
 - Musescore 4.0.0

**Additional context**

### PR DESCRIPTION
**Pull request by HemantAntony**
Resolves: #15667 
Resolves: #15654
Resolves: #15666

- [x] I signed the [CLA](https://musescore.org/en/cla)
- [x] The title of the PR describes the problem it addresses
- [x] Each commit's message describes its purpose and effects, and references the issue it resolves
- [x] If changes are extensive, there is a sequence of easily reviewable commits
- [x] The code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
- [x] There are no unnecessary changes
- [x] The code compiles and runs on my machine, preferably after each commit individually
- [x] I created a unit test or vtest to verify the changes I made (if applicable)

### CODE REVIEW

File: src/languages/languagestypes.h [hunk 1]
```diff
@@ -38,6 +38,7 @@ struct Language
 {
     QString code;
     QString name;
+    Qt::LayoutDirection rtl = Qt::LeftToRight; // right to left
```
Comment (Jojo-Schmitz) [inline src/languages/languagestypes.h:41]: Setting and comment don't match

Comment (HemantAntony) [reply src/languages/languagestypes.h:41]: I meant the full form of rtl - right to left. But I see your point. I think I'll delete the comment

Comment (igorkorsukov) [reply src/languages/languagestypes.h:41]: I think better name  just `direction`

Comment (Jojo-Schmitz) [reply src/languages/languagestypes.h:41]: I agree

**Comment by HemantAntony** [comment]
This PR solves three issues #15654, #15666, #15667

**Comment by Tantacrul** [comment]
Tested on MacOS. All three issues look resolved by this fix. @cbjeukendrup / @RomanPudashkin / @Eism, can you review and merge when you find time?

**Review by igorkorsukov — APPROVED** [review]

**Pull request merged**